extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/Jose4jRecorder.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.oidc.runtime;
+
+import org.jose4j.jwa.AlgorithmFactoryFactory;
+
+import io.quarkus.runtime.annotations.Recorder;
+
+@Recorder
+public class Jose4jRecorder {
+
+    public void initialize() {
+        AlgorithmFactoryFactory.getInstance();
```

Comment:
This uses ` System.getProperty` which should never be used to configure behavior at native-image build time.

To be fair, it seems to use it only for logging, but it makes suspicious of what else this method does and it absolutely needs to be checked thoroughly

Reply:
> The rest of the method is sound, these properties are not having any impact.

If you have checked it thoroughly, that's fine with me.

Reply:
@geoand Yeah, I searched if any of those properties can be seen somewhere else in that code :-). The rest is just loading supported algorithms, none of them would need a version/vendor/javahome property. I suppose, purely for debugging purposes, seeing Java version and vendor can be useful in context of the Jose4 initialization as a support for various algorithms may vary...
Agreed it was not really essential though